### ISSUE TITLE
VSA dev intent: Learning center, MVP 1.1 - media_list_images

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#14077]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This issue contains the VSA accessibility feedback for the Learning center, MVP 1.1 - media_list_images

#### Desktop

![Screen Shot 2020-09-25 at 2 51 26 PM](https://user-images.githubusercontent.com/57469/94305058-9c168c00-ff3e-11ea-94cc-508215a93f5d.png)

#### Mobile

_Not provided_

## Point of Contact

**VFS Point of Contact:** Jennifer

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jenstrickland** [comment]
Accessibility issues shown in the CMS preview should be addressed.

![Screen Shot 2020-09-25 at 2 43 41 PM](https://user-images.githubusercontent.com/57469/94305099-acc70200-ff3e-11ea-8d4e-40486f2476ae.png)

**Comment by jenstrickland** [comment]
![Screen Shot 2020-09-25 at 2 47 20 PM](https://user-images.githubusercontent.com/57469/94305171-c7997680-ff3e-11ea-81e1-faa7f03066e4.png)

### Current code
```
<!-- Images list -->
<ul class="usa-unstyled-list">
  <li classname="vads-u-display--flex vads-u-margin-bottom--2">
    <div class="vads-1-grid-container vads-u-padding-x--0 vads-u-margin-y--2">
      ::before
      <div class="vads-1-row">
        <div class="vads-1-col--2">
          <img class="vads-u-border--1px vads-u-border-color--gray-light" alt="01 Latin cross" height="566" width="480" src="http://staging.cms.va.gov/sites/default/files/2020-08-01%20Latin%20cross.jpg">
        </div>
        <div class="vads-1-col--10 vads-u-padding-left--2">
          <h2 class="vads-u-font-size--h3" id="01-latin-cross">01 Latin Cross</h2>
        </div>
        <div class="vads-1-col--10 vads-u-padding-left--2">
          <h2 class="vads-u-font-size--h3" id="01-latin-cross">01 Latin Cross</h2>
        </div>
      </div>
      ::after
<!-- etcetera -->
```

### Recommendation

* **Must** add role for the list and listitems; **why:** when CSS `list-style:none` is used, Safari needs the semantics added back

* The alt text **should** not simply repeat the label; **why:** it would be more helpful to describe the image in the alt text, as non-sighted users need to know the description and will get the label already. While CMS authors are unlikely to know this and are bound to make mistakes, we can provide best practice behavior modeling.

* Height and width styling **should** happen in the CSS; **why:** by hard-coding the dimensions in the HTML responsive behavior is impacted. Do we tend not to close image tags on VA.gov? @andaleliz, @rtwell  — is there a responsive design for the media_list?

* Change the `h2` to a `dfn`; **why:** a heading level is too heavy-handed here and will introduce a lot of extra noise for screen reader users; the definition element will add a label and the utility class can be added to get the styling matching the visual design.

```diff
<!-- Images list -->
+ <ul class="usa-unstyled-list" role="list">
+   <li classname="vads-u-display--flex vads-u-margin-bottom--2" role="listitem">
    <div class="vads-1-grid-container vads-u-padding-x--0 vads-u-margin-y--2">
      ::before
      <div class="vads-1-row">
        <div class="vads-1-col--2">
+          <img class="vads-u-border--1px vads-u-border-color--gray-light" alt="dark outline of a Latin cross" src="http://staging.cms.va.gov/sites/default/files/2020-08-01%20Latin%20cross.jpg" />
        </div>
        <div class="vads-1-col--10 vads-u-padding-left--2">
+          <dfn class="vads-u-font-size--h3" id="01-latin-cross">01 Latin Cross</dfn>
        </div>
      </div>
      ::after
<!-- etcetera -->
```

**Comment by andaleliz** [comment]
>@andaleliz, @rtwell — is there a responsive design for the media_list?

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/11497

**Comment by jenstrickland** [comment]
@andaleliz @MickinSahni I'm not sure the media_list_images work should begin in development until we have the mobile design documented, so that development can take a mobile-first approach and make sure to plan accordingly (such as not putting dimensions in the HTML and other styling). Welcome your thoughts!

**Comment by MickinSahni** [comment]
Hi! Any work we did on 1.1 capabilities (e.g. Images, Videos, Checklist) was around laying the foundation to iterate on. 

We have mobile design work on 1.1 in upcoming sprints after which we'll return to build considerations e.g. dev intent for 1.1 items. 

We're focused right now on building 1.0 of the LC, which images are **not** part of.

**Comment by jenstrickland** [comment]
Apologies, I'm not sure what items I should provide dev intent feedback on.

**Comment by kelsonic** [comment]
Design has changed a lot for this template since this ticket was created, closing but feel free to reopen if needed @brianalloyd

**Comment by brianalloyd** [comment]
@kelsonic concur, will transition any future workflow to new ticket.  Appreciate you closing in support.
